backend/src/main/java/com/jobwait/persistence/PostgresController.java

```diff
@@ -6,16 +6,39 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
 
+import com.jobwait.domain.Answer;
 import com.jobwait.domain.Answers;
 import com.jobwait.domain.User;
 import com.jobwait.persistence.adapters.PostgresUserAdapter;
+import com.jobwait.persistence.adapters.AnswerAdapter;
 
 public class PostgresController extends PersistenceController {
     private String jdbcUrl = "jdbc:postgresql://localhost:5432/mydatabase";
     private String dbUser = "postgres";
     private String dbPassword = "password";
 
+    private final String answerValueQuestionString = Answers.listOfTypeOfAnswers.stream().map((answerString) -> "?")
+            .collect(Collectors.joining(", "))
+            .concat(", ?");// concat for userid
+
+    private final String answerValueExcludedUpdateString = Answers.listOfTypeOfAnswers.stream()
+            .map((answerString) -> answerString + " = " + "EXCLUDED."
+                    + answerString)
+            .reduce(
+                    (answerString, answerString2) -> answerString + " , "
+                            + answerString2)
+            .orElseThrow();
+
+    private final String answerValueReturnString = Answers.listOfTypeOfAnswers.stream()
+            .map(answerString -> answerString + " AS " + answerString)
+            .reduce((answerString, answerString2) -> answerString + " , "
+                    + answerString2)
+            .orElseThrow();
```

Comment:
i still can't read this, i hope it works

Reply:
what can i do to make it more readable, ill have to think about this, its not crucial so if i come up with a nice idea ill include it in the next PR. your concerns have not gone unheard.

Reply:
honestly my eyes just glaze over when I see these 3 chunks. if it came down to maintaining this, i'd just force myself to understand it, so it's likely a skill issue. In any case, it's better than what it was before you moved them to variables.